java/clients/src/test/java/sleeper/clients/admin/testutils/AdminClientMockStoreBase.java

```diff
@@ -67,9 +72,24 @@ public void startClient(AdminClientStatusStoreFactory statusStores, QueueMessage
                 .start(instanceId);
     }
 
-    protected void setInstanceTables(InstanceProperties instanceProperties, String... tableNames) {
+    protected void setInstanceTables(InstanceProperties instanceProperties, TableStatus... tableIds) {
         setInstanceProperties(instanceProperties);
-        when(store.listTables(instanceProperties.get(ID))).thenReturn(Arrays.asList(tableNames));
+        List.of(tableIds).forEach(tableIndex::create);
```

Comment:
It seems like this could delegate to saveTableProperties like the other methods below do?

Reply:
Fixed, thanks